broker/setup_broker/setup_broker.sh

```diff
@@ -129,4 +129,28 @@ fi
 #--- Deploy Cloud Functions
 echo
 echo "Configuring Cloud Functions..."
-./deploy_cloud_fncs.sh "${testid}" "${teardown}" "${survey}" "${versiontag}" "${zone}"
+cd .. && cd cloud_functions
```

Comment:
I started some bad trends in these shell scripts and `cd`ing around a lot is one of them. Unfortunately it's not easy to fix here because the cloud function gets deployed from the current working directory, so we need to `cd` into it. We could avoid the `cd` by using the `--source` flag in the [`gcloud functions deploy` command](https://cloud.google.com/sdk/gcloud/reference/functions/deploy#--source), but it would have to be set in each `deployment.sh` script and be relative to whatever the current working directory is -- that is generally `broker/setup_broker` before we start `cd`ing around, but that seems like a weird, brittle, relative path to set in those files.

Codacy is complaining about this, so for now I think we should fix it by just accepting Codacy's suggestion and doing 

```suggestion
cd .. && cd cloud_functions || exit
```

and doing similar in all the `cd`s that follow.